homeassistant/core.py

```diff
@@ -1127,6 +1127,28 @@ def execute_service() -> None:
             _LOGGER.exception('Error executing service %s', service_call)
 
 
+class ApiConfig:
```

Comment:
Does it need to be in core? I think that it can just live in the http component

Reply:
Moved. Not sure how to set typing for config.api, use Any for now.